src/lib/code-sample/go.ts

```diff
@@ -0,0 +1,167 @@
+import { pascalCase } from 'change-case'
+
+import type { Json, NonNullJson } from 'lib/json.js'
+
+import { createJsonResponse } from './create-json-response.js'
+import type { CodeSampleDefinition, Context } from './schema.js'
+
+const DEFAULT_GO_PACKAGE_NAME = 'api'
+const GO_PACKAGE_BASE_PATH = 'github.com/seamapi/go'
```

Comment:
```suggestion
const defaultGoPackageName = 'api'
const goPackageBasePath = 'github.com/seamapi/go'
```

Reply:
We should actually make these config options for blueprint. But ok as consts now.